.scalafmt.conf

```diff
@@ -20,4 +20,3 @@ newlines.source = keep
 project.git = true
 
 runner.dialect = scala213
-
```

Comment:
Let's bump the `version` to 3.8.4-RC1, that comes with support for recognizing `.mill` files.

Also, it seems like ScalaFmt does support per-glob overrides https://scalameta.org/scalafmt/docs/configuration.html#fileoverride, so I think we should be able to use that to enable `docstrings.style = keep` for `example/**/build.mill` without needing to enable it for the other Scala code

Reply:
Oh nice, I didn't quickly find how to apply file overrides like that! Thanks!